Dockerfile

```diff
@@ -1,28 +1,24 @@
 # stretch required for gc-stats post build
-FROM node:16.14.2-bullseye AS builder
```

Comment:
Done

Reply:
why do you want to use Debian 10 (buster) instead of already set Debian 11 (bulleye) ?

[Debian version reminder](https://wiki.debian.org/DebianReleases)

Version | Code name | Release date | End of life date | EOL LTS | EOL ELTS
-- | -- | -- | -- | -- | --
14 | Forky |   |   |  
13 | Trixie |   |   |  
12 | Bookworm |   |   |  
11 | Bullseye | 2021-08-14 | ~2024-07 |  
10 | Buster | 2019-07-06 | 2022-09-10
9 | Stretch | 2017-06-17 | 2020-07-18 | 2022-07-01 | 2027-06-30
8 | Jessie | 2015-04-25 | 2018-06-17 | 2020-06-30 | 2025-06-30

Reply:
It's a mistake.  
Done